utils/gui/Minimap.js

```diff
@@ -0,0 +1,171 @@
+import PlanarView from 'Core/Prefab/PlanarView';
+import Coordinates from 'Core/Geographic/Coordinates';
+import { CAMERA_TYPE } from 'Renderer/Camera';
+import { MAIN_LOOP_EVENTS } from 'Core/MainLoop';
+import Widget from './Widget';
+
+
+const DEFAULT_OPTIONS = {
+    minPixelSize: 2000,
+    maxPixelSize: Infinity,
+    size: 150,
+    position: 'bottom-left',
+};
+
+
+/**
+ * A widget for minimap
+ *
+ * @property    {HTMLElement}   domElement      An html div containing the minimap.
+ * @property    {HTMLElement}   parentElement   The parent HTML container of `this.domElement`.
+ */
+class Minimap extends Widget {
+    /**
+     * @param   {View}                  view                                    The iTowns view the minimap should be
+                                                                                * linked to.
+     * @param   {ColorLayer}            layer                                   The {@link ColorLayer} that should be
+                                                                                * displayed on the minimap.
+     * @param   {Object}                [options]                               The minimap optional configuration.
+     * @param   {HTMLElement}           [options.parentElement=view.domElement] The parent HTML container of the div
+                                                                                * which contains minimap widgets.
+     * @param   {number}                [options.minPixelSize=2000]             The minimal size in meters a pixel of
+                                                                                * the minimap can reach.
+     * @param   {number}                [options.maxPixelSize=Infinity]         The maximal size in meters a pixel of
```

Comment:
I think it's better Scale instead of pixel size

Reply:
I replaced by Scale